[2022-03-19 23:06:21,044] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: udac_new_v1_dag.task_Dimension_table.load_users_dim_table manual__2022-03-19T15:03:21.401193+00:00 [queued]>
[2022-03-19 23:06:21,061] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: udac_new_v1_dag.task_Dimension_table.load_users_dim_table manual__2022-03-19T15:03:21.401193+00:00 [queued]>
[2022-03-19 23:06:21,062] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-19 23:06:21,063] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-03-19 23:06:21,063] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-19 23:06:21,146] {taskinstance.py:1264} INFO - Executing <Task(LoadDimensionOperator): task_Dimension_table.load_users_dim_table> on 2022-03-19 15:03:21.401193+00:00
[2022-03-19 23:06:21,150] {standard_task_runner.py:52} INFO - Started process 52993 to run task
[2022-03-19 23:06:21,158] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'udac_new_v1_dag', 'task_Dimension_table.load_users_dim_table', 'manual__2022-03-19T15:03:21.401193+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/udac_v1.py', '--cfg-path', '/var/folders/lh/ngst1lt51m9_gyz7qrg2f3nh0000gn/T/tmp91g2l1_p', '--error-file', '/var/folders/lh/ngst1lt51m9_gyz7qrg2f3nh0000gn/T/tmpr1gs2mxn']
[2022-03-19 23:06:21,161] {standard_task_runner.py:77} INFO - Job 141: Subtask task_Dimension_table.load_users_dim_table
[2022-03-19 23:06:21,310] {logging_mixin.py:109} INFO - Running <TaskInstance: udac_new_v1_dag.task_Dimension_table.load_users_dim_table manual__2022-03-19T15:03:21.401193+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2022-03-19 23:06:21,522] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=udacity
AIRFLOW_CTX_DAG_ID=udac_new_v1_dag
AIRFLOW_CTX_TASK_ID=task_Dimension_table.load_users_dim_table
AIRFLOW_CTX_EXECUTION_DATE=2022-03-19T15:03:21.401193+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-19T15:03:21.401193+00:00
[2022-03-19 23:06:21,523] {load_dimension.py:83} INFO - Starting to truncate users
[2022-03-19 23:06:21,542] {base.py:70} INFO - Using connection to: id: redshift. Host: redshift-cluster-1.cvmsgkpkbsh2.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: ***, extra: {}
[2022-03-19 23:06:22,693] {dbapi.py:225} INFO - Running statement: 
        TRUNCATE TABLE "public"."users"
        , parameters: None
[2022-03-19 23:06:23,232] {load_dimension.py:93} INFO - Starting to insert users
[2022-03-19 23:06:23,247] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-03-19 23:06:23,249] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
[2022-03-19 23:06:23,251] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2022-03-19 23:06:23,253] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2022-03-19 23:06:23,254] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
[2022-03-19 23:06:23,255] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-19 23:06:23,256] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
[2022-03-19 23:06:23,258] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
[2022-03-19 23:06:23,259] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
[2022-03-19 23:06:23,260] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-19 23:06:23,262] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
[2022-03-19 23:06:23,263] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/***_project/lib/python3.9/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-19 23:06:23,306] {base.py:70} INFO - Using connection to: id: redshift. Host: redshift-cluster-1.cvmsgkpkbsh2.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: ***, extra: {}
[2022-03-19 23:06:24,448] {dbapi.py:225} INFO - Running statement: 
        INSERT INTO "public"."users" (userid, first_name, last_name, gender, level)
 
        SELECT distinct userid, firstname, lastname, gender, level
        FROM staging_events
        WHERE page='NextSong'
    
        , parameters: None
[2022-03-19 23:06:24,979] {dbapi.py:233} INFO - Rows affected: 104
[2022-03-19 23:06:25,609] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=udac_new_v1_dag, task_id=task_Dimension_table.load_users_dim_table, execution_date=20220319T150321, start_date=20220319T150621, end_date=20220319T150625
[2022-03-19 23:06:25,641] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-19 23:06:25,691] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
